tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -872,6 +875,10 @@ public function testCreateDeployment()
         $this->assertEquals('waiting', $deployment['body']['status']);
         $this->assertEquals(true, (new DatetimeValidator())->isValid($deployment['body']['$createdAt']));
 
+        $rule = $this->getRule($rule['body']['$id']);
+        $this->assertEquals(200, $rule['headers']['status-code']);
+        $this->assertEquals($deployment['body']['$id'], $rule['body']['deploymentId']);
```

Comment:
Let's add E2E test with router too please, sending request to domain, expecting specific mesasge in body (like "wait until build finished" or such)